give.php

```diff
@@ -356,6 +356,8 @@ private function setup_constants() {
 		private function includes() {
 			global $give_options;
 
+			require_once ABSPATH . 'wp-admin/includes/plugin.php';
```

Comment:
@mehul0810 can we wrap file require:
```
if( ! funciton_exists('is_plugin_active') ){
    require_once ABSPATH . 'wp-admin/includes/plugin.php';
}
```

It is more safe to load file this way.